README.md

```diff
@@ -130,6 +141,32 @@ async function increaseRepeatedly(base: number): Promise<number[]> {
 }
 ```
 
+When `use_static_classes` is set to false, you can implement your code will look like this:
+
+```ts
+import { increase } from "./counter.pb";
```

Comment:
You could add a third example here suggesting a way to get an equivalent to the old format, if desired:

```ts
import { * as CounterService } from "./counter.pb";
```

(I think this works, right?)

Reply:
That should work, I think you'd also get `CounterService.CounterServiceClient`